sdk/ai/azure-ai-client/samples/agents/sample_agents_stream_eventhandler_with_toolset.py

```diff
@@ -61,8 +62,8 @@
 
 class MyEventHandler(AgentEventHandler):
 
-    def __init__(self, client: Agent):
-        self._client = client
+    def __init__(self, agents: AgentsOperations):
```

Comment:
what is the reason for this change?

Reply:
Oh.   I found that this sample doesn't work after migration.   The function, submit_tool_outputs_to_run is in the AgentOperations.   Not in the Agent object.

Reply:
ok, thanks!